cluster/kubemark/gce/config-default.sh

```diff
@@ -22,93 +22,20 @@
 # shellcheck disable=SC2034 # Variables sourced in other scripts.
 
 source "${KUBE_ROOT}/cluster/gce/config-common.sh"
+NUM_NODES="${KUBEMARK_NUM_NODES:-}"
 
-GCLOUD=gcloud
-ZONE=${KUBE_GCE_ZONE:-us-central1-b}
-REGION=${ZONE%-*}
-NUM_NODES=${KUBEMARK_NUM_NODES:-10}
-NUM_WINDOWS_NODES=${KUBEMARK_NUM_WINDOWS_NODES:-0}
-MASTER_SIZE=${KUBEMARK_MASTER_SIZE:-n1-standard-$(get-master-size)}
-MASTER_DISK_TYPE=pd-ssd
-MASTER_DISK_SIZE=${MASTER_DISK_SIZE:-$(get-master-disk-size)}
-MASTER_ROOT_DISK_SIZE=${KUBEMARK_MASTER_ROOT_DISK_SIZE:-$(get-master-root-disk-size)}
-REGISTER_MASTER_KUBELET=${REGISTER_MASTER:-false}
-PREEMPTIBLE_NODE=${PREEMPTIBLE_NODE:-false}
-NODE_ACCELERATORS=${NODE_ACCELERATORS:-""}
-CREATE_CUSTOM_NETWORK=${CREATE_CUSTOM_NETWORK:-false}
-EVENT_PD=${EVENT_PD:-false}
-
-MASTER_OS_DISTRIBUTION=${KUBE_MASTER_OS_DISTRIBUTION:-gci}
-NODE_OS_DISTRIBUTION=${KUBE_NODE_OS_DISTRIBUTION:-gci}
-MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-cos-beta-73-11647-64-0}
-MASTER_IMAGE_PROJECT=${KUBE_GCE_MASTER_PROJECT:-cos-cloud}
 CLEANUP_KUBEMARK_IMAGE=${CLEANUP_KUBEMARK_IMAGE:-true}
 
-# GPUs supported in GCE do not have compatible drivers in Debian 7.
-if [[ "${NODE_OS_DISTRIBUTION}" == "debian" ]]; then
-    NODE_ACCELERATORS=""
-fi
-
-NETWORK=${KUBE_GCE_NETWORK:-e2e}
-if [[ "${CREATE_CUSTOM_NETWORK}" == true ]]; then
-  SUBNETWORK="${SUBNETWORK:-${NETWORK}-custom-subnet}"
-fi
-INSTANCE_PREFIX="${INSTANCE_PREFIX:-"default"}"
-MASTER_NAME="${INSTANCE_PREFIX}-kubemark-master"
-AGGREGATOR_MASTER_NAME="${INSTANCE_PREFIX}-kubemark-aggregator"
-MASTER_TAG="kubemark-master"
-EVENT_STORE_NAME="${INSTANCE_PREFIX}-event-store"
-MASTER_IP_RANGE="${MASTER_IP_RANGE:-10.246.0.0/24}"
-CLUSTER_IP_RANGE="${CLUSTER_IP_RANGE:-$(get-cluster-ip-range)}"
-RUNTIME_CONFIG="${KUBE_RUNTIME_CONFIG:-}"
-TERMINATED_POD_GC_THRESHOLD=${TERMINATED_POD_GC_THRESHOLD:-100}
-KUBE_APISERVER_REQUEST_TIMEOUT=300
-ETCD_COMPACTION_INTERVAL_SEC="${KUBEMARK_ETCD_COMPACTION_INTERVAL_SEC:-}"
```

Comment:
I think we need this one (IIRC we're overwriting it in 5k-kubemark job).

Reply:
I moved this (and all KUBEMARK_* envs) to start-master func.